activities_sticky_queues/starter.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+from uuid import uuid4
+
+from temporalio.client import Client
+
+from activities_sticky_queues.tasks import FileProcessing
+
+
+async def main():
+    # Connect client
+    client = await Client.connect("localhost:7233")
+
+    # Start 10 concurrent workflows
+    futures = []
+    for _ in range(10):
+        handle = client.execute_workflow(
+            FileProcessing.run, id=str(uuid4()), task_queue="distribution-queue"
+        )
```

Comment:
```suggestion
    for _ in range(10):
        handle = client.execute_workflow(
            FileProcessing.run,
            id=f"activity_sticky_queue-workflow-id-{i}",
            task_queue="activity_sticky_queue-distribution-queue",
        )
```

Fixed workflow IDs and qualify the task queue with the sample name (matches our other samples)